frontend/src/Components/Basket.js

```diff
@@ -64,18 +71,28 @@ export default function Basket() {
       })
       .catch(function (error) {
         console.log(error);
+        alert(error);
       });
   };
 
+  function handlePayment() {
+    //indicates that user is paying by card
+    setCardPayment(true);
+
+    console.log(cardPayment);
```

Comment:
It's best to clean up `console.log` calls before committing - add too many and it becomes difficult to find any info amongst the noise.